### PR TITLE
[Rust] Fix gotham

### PR DESCRIPTION
**Pull request by waghanza**

### CODE REVIEW

File: rust/gotham/Cargo.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ authors = ["Isaac Whitfield <iw@whitfin.io>"]
 [dependencies]
 gotham = "~0.5"
 gotham_derive = "~0.5"
-hyper = "*"
+hyper = "~0.13"
```
Comment (msrd0) [inline rust/gotham/Cargo.toml:9]: I strongly recommend you remove this dependency and use the `gotham::hyper` re-export instead

Comment (waghanza) [reply rust/gotham/Cargo.toml:9]: and is it the same for `serde` ?

Comment (msrd0) [reply rust/gotham/Cargo.toml:9]: I'd recommend not using `*` as the version, but the serde version is independent from gotham, so it shouldn't be that much of a problem.

Comment (waghanza) [reply rust/gotham/Cargo.toml]: so what do you recommend ? using a range constraint like `~1.0` ?

Comment (msrd0) [reply rust/gotham/Cargo.toml]: Just put `serde = "1.0"`, cargo will treat that as `>= 1.0, < 2.0` so that should be fine

**Pull request merged**